experimental/virtio/src/queue/virtq.rs

```diff
@@ -122,6 +125,8 @@ pub struct UsedRing<const QUEUE_SIZE: usize> {
     pub idx: u16,
     /// The ring-buffer containing the used elements.
     pub ring: [UsedElem; QUEUE_SIZE],
+    /// Event details. Only used if VIRTIO_F_EVENT_IDX has been negotiated.
```

Comment:
I am not sure I understand "Only used if ...". Does it mean I have to make sure "VIRTIO_EVENT" has been negotiated before I use the `avail_event`? (Similar in line 99)

Reply:
The field is not used in our implementation, but is there as padding. Reading or writing it will have no effect. Updated the comment to clarify.